packages/excavator-script/src/projects/mummingTrunk.ts

```diff
@@ -0,0 +1,144 @@
+    ],
+    flying: [
+      "From <pronoun-pos> position above the battlefield, <name> helps you get the jump on your foe in typical Oliver Cromwell style\\.",
+      "<name> shouts some intel about enemy troop formations down at you from <pronoun-pos> position over the battlefield\\. Because that's something Oliver Cromwell did\\.",
+      "<name> dissolves the Rump Parliament and is elected Lord Protector of England by your other familiars\\. Also, you win initiative\\.",
+      "<name> waves her saber in the air and orders a cavalry charge, taking your foe by surprise\\.",
+    ],
+  },
+  "6": {
+    mechanical: [
+      "<name>, dressed as The Doctor, gives you a quick MRI\\.",
+      "<name> consults some bleeping Doctor-type machines and writes you a prescription\\.",
+      "Doctor <name> puts a stick in your mouth and shines a light-up gadget in your eyes\\. For some reason, you feel better\\.",
+      "<name>, dressed as The Doctor, jumps into <pronoun-pos> time machine and\\.\\.\\. what\\? Just a regular doctor\\? Oh\\.",
+      "\\.\\.\\.I guess <pronoun-subj> just gives you some pills or something then\\.",
+    ],
+    evil: [
+      "Doctor <name> writes your opponent a prescription for <number> cc of damage, delivered anally\\.",
+      "Doctor <name> gets an evil glint in <pronoun-pos> eye, and another in <pronoun-pos> scalpel, and <number> of them in your foe's arm\\.",
+      "<name> puts a pinky \\(or pinky-analogue\\) to the corner of <pronoun-pos> mouth and sneers something about a billion Meat\\. Your opponent sighs so heavily, they cause <number> damage to themself\\.",
+      "With an evil grin, <name> pulls a wicked-looking medical instrument out of the coat of <pronoun-pos> Doctor costume and jabs your opponent for <number> damage\\.",
+    ],
+  },
+  "7": {
+    sleazy: [
+      "<name> tells you a joke so dirty, you're pretty sure even Miss Funny wouldn't stand for it\\.",
+      "<name>, in <pronoun-pos> Miss Funny costume, tells you a joke so filthy you barely even understand it\\.",
+      '<name> tells you Miss Funny\'s version of "The Aristocrats", which is way filthier than you might expect\\.',
+      "<name>, dressed as Miss Funny, tells an extremely bawdy joke\\. Far too bawdy to include here\\. Just\\.\\.\\. very very bawdy\\.",
+    ],
+    bug: [
+      "<name> plays a great Miss Funny-style trick on your opponent, filling their pants with live bees\\. <elemental> damage worth of hilarity ensues\\.",
+      "<name> plays a hilarious Miss Funny prank on your foe, laying a clutch of eggs in <pronoun-pos> mouth and dealing <elemental> damage\\.",
+      "<name> vomits corrosive stomach secretions onto your opponent, causing <elemental> damage\\. What a hilarious prank!",
+      "<name> tricks your foe by summoning a swarm of greasy locusts to devour them for <elemental> damage\\. Guess that's why they call <pronoun-obj> Miss Funny!",
+    ],
+  },
+};
+
+export const MUMMING_TRUNK: ExcavatorProject = {
```

Comment:
Likely an unnecessary project per @SSBBHax's heroic efforts

Reply:
Because this one checks whether its match is present in the proxy record before sending a datapoint, I think this is non-chatty and thus still useful.